contain_learning/Dockerfile

```
@@ -0,0 +1,5 @@
+FROM centos:latest
+RUN yum install python36 -y
```

Identified a potential vulnerability, here’s a suggested change for The 'yum install' command does not clean up the package manager cache, which can lead to larger image sizes.:

```suggestion
RUN yum install python36 -y && yum clean all
```